lib/gulp-tasks/file-globbing.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
I suggest to have pattern module names like `gulp-<task-name>-task`, so we know it contains gulp tasks.
`gulp-glob-inject-task`?